### DISCUSSION TITLE
Data status in acquired .mha file

### DISCUSSION THREAD

**Discussion by QiLi111** [PlusToolkit/PlusLib#908] [Q&A]
Hi,

Can I ask a question about the acquired .mha file? Thanks.

I have defined the tool I used as 'Probe' in the config file, and only use this tool to acquire data. Therefore, the other two definitions is not used in my experiment and the no data is acquired.

Previously, the status of other tools that called 'reference' and 'Stylus' are all 'INVALID', and the corresponding 'ReferenceToTrackerTransform' is ' -1.#IND'. But recently I have found that the status of other tools are all 'MISSING', and the corresponding 'ReferenceToTrackerTransform' is '-nan(ind)'. Please see below for more detailed information.

Previously: 
```
Seq_Frame0002_ProbeToTrackerTransform = -0.6085459230420159 0.1818039078607282 0.7724112695661648 62.64445495605469 -0.784372163168218 -0.2851844142708686 -0.5508448954482772 19.7923583984375 0.1201339337199792 -0.9410723237453627 0.3161497268941315 -1013.365295410156 0 0 0 1  

Seq_Frame0002_ProbeToTrackerTransformStatus = INVALID 

Seq_Frame0002_ReferenceToTrackerTransform = -1.#IND -1.#IND -1.#IND 0 -1.#IND -1.#IND -1.#IND 0 -1.#IND -1.#IND -1.#IND 0 0 0 0 1  

Seq_Frame0002_ReferenceToTrackerTransformStatus = INVALID 

Seq_Frame0002_StylusToTrackerTransform = -1.#IND -1.#IND -1.#IND 0 -1.#IND -1.#IND -1.#IND 0 -1.#IND -1.#IND -1.#IND 0 0 0 0 1  

Seq_Frame0002_StylusToTrackerTransformStatus = INVALID 

Seq_Frame0002_Timestamp = 34.45044285714285 

Seq_Frame0002_ImageStatus = OK 
```

Rencently:
```
Seq_Frame0002_ProbeToTrackerTransform = -0.7976552612131267 -0.5709319223959501 -0.1943780446662716 92.90617786637084 -0.5137998008519967 0.8120516326789634 -0.276734368136446 -59.99113454921181 0.3158414933213601 -0.1208672240629955 -0.9410819652101716 -750.3170142262363 0 0 0 1  

Seq_Frame0002_ProbeToTrackerTransformStatus = OK 

Seq_Frame0002_ReferenceToTrackerTransform = -nan(ind) -nan(ind) -nan(ind) 0 -nan(ind) -nan(ind) -nan(ind) 0 -nan(ind) -nan(ind) -nan(ind) 0 0 0 0 1  

Seq_Frame0002_ReferenceToTrackerTransformStatus = MISSING 

Seq_Frame0002_StylusToTrackerTransform = -nan(ind) -nan(ind) -nan(ind) 0 -nan(ind) -nan(ind) -nan(ind) 0 -nan(ind) -nan(ind) -nan(ind) 0 0 0 0 1  

Seq_Frame0002_StylusToTrackerTransformStatus = MISSING 

Seq_Frame0002_Timestamp = 5305.566000000004 

Seq_Frame0002_ImageStatus = OK 
```

I don't know why they have been changed. Could you please tell me whether there are some parameters that I accidentally modified? Does this affected the data acquired from 'Probe', which is for further use.

Thanks for your help very much.

Best regards,
Qi

**Comment by Sunderlandkyl** [comment]
I don't think the transform data should be affected. Can you give some background on what version of Plus was used to generate each file?

**Reply by QiLi111** [reply]
Hi, 
Thanks for your reply very much.

The version of Plus I am now using is 'Plus Application 2.8.02019-11-05 (Win32)'. The laptop is Dell Precision M4800 - 64 bit, windows 10.

Thanks again!

Best regards,
Qi
